src/core.js

```diff
@@ -25,6 +31,14 @@ jQuery.fn.init = function( arg1 ) {
 };
 jQuery.fn.init.prototype = jQuery.fn;
 
+jQuery.htmlPrefilter = function( html ) {
```

Comment:
I think this code should lie in `src/manipulation.js` as it was in jQuery. In that case you won't need a separate `rxhtmlTag` definition as it's already defined there.

Reply:
I'll move it. I'd actually forgotten we had it there.